.github/workflows/e2e-prometheus3.yaml

```diff
@@ -0,0 +1,105 @@
+name: e2e-prometheus3
+on:
+  schedule:
+    - cron:  '30 14 * * *' # Every day 14:30
+jobs:
+  e2e-tests:
```

Comment:
If possible it would be good to share the same workflow definition between PR and periodic via some parameter.

Reply:
No rather that we could look at [reusable workflows](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#creating-a-reusable-workflow) to minimize the duplication between the 2 types of jobs.

Reply:
I can update the workflow file separate as I need to look into reusable workflow working. 
Are the other changes ok?

Reply:
https://github.com/prometheus-operator/prometheus-operator/issues/6948